src/App/Fossa/Main.hs

```diff
@@ -216,7 +221,13 @@ hiddenCommands =
           ( info
               (ContainerCommand <$> containerOpts)
               (progDesc "Run in Container Scan mode")
-    )
+          )
+        <> command
+          "compatibility"
+          ( info
+              (CompatibilityCommand <$> compatibilityOpts)
+              (progDesc "Run fossa cli v1. Supply arguments as \"fossa compatibility -- analyze --project test\"")
```

Comment:
Are we allowing people to run any v1 command here?  Is that why we're making this change?

Reply:
Correct, the main goal is to let them run archive scanning, but by allowing them to run anything we prevent ourselves from getting hung up in the future.

Reply:
I don't know if I agree with this.  I think I'd rather only allow certain options, or people will just continue using v2 as v1 in other scenarios.  I think this makes it much harder to deprecate v1 eventually.

Reply:
Maybe it's reasonable to force the existence of `analyze` and whichever option triggers archive scanning.

Reply:
I think forcing `fossa analyze` is reasonable. I really don't have a strong feeling about this, but I would prefer to be more restrictive to start, so I'll enforce this.